javascript/javascript1/week1/Goodboy-Oldboy.js

```diff
@@ -0,0 +1,10 @@
+const dogYearOfBirth = 2010;
+const dogYearFuture = 2028;
+const dogYear = dogYearFuture - dogYearOfBirth;
+const shouldShowResultInDogYears = false;
+if (shouldShowResultInDogYears == true) {
```

Comment:
That's correct. As option it can be written in a bit shorter way
```javascript
if (shouldShowResultInDogYears) {
   console.log("bla-bla-bla")
} else {
   console.log("bla-bla-bla")
}
```